data/test/choices/judges.csv

```diff
@@ -2,7 +2,7 @@ name,email,gender,phone,institution,adj_core,independent,test_score,institution_
 Porro Goold,PorroGoold@einrot.com,other,+49 0221 49 63 43,Mauville,T,no,5,,,
 Simone Grunewald,SimoneGrunewald@jourrapide.com,femAle ,+31 06-84985326,Celadon,TRUE , FALSE ,5,,,
 Satoe Murai,SatoeMurai@einrot.com,female,+61 (08) 8295 0262,Goldenrod,true,FALSE,5,,,Itala Ricci
-Mariko Oonishi,MarikoOonishi@jourrapide.com,f,+1 716-859-2311,Lilycove,FALSE,  TRUE,3,,,
+Mariko Oonishi,MarikoOonishi@jourrapide.com,female,+1 716-859-2311,Lilycove,FALSE,  TRUE,3,,,
```

Comment:
This is actually supposed to be just `f`: the point of the file is to check that the importer can accept a variety of different formats. Has this line been failing for you?